ext/lib/crypto/mbedtls/configs/config-full-mqtt.h

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Could you separate the JWT part from this sample, meaning that submitting the JWT in separate PR?

Reply:
I would rather see the new MQTT at #10125 used as a base and this sample building on top of that PR. We have already a MQTT library that is being replaced by #10125 so I would not like to introduce yet another MQTT library even thou this is a sample.

Reply:
I am not a fan of separating work that is required by the work I actually want to do.  github really doesn't handle this well.  If we could get immediate turnaround on JWT, that would be a compelling argument.  But in most cases this draws out things considerably.

Reply:
and I'd be happy to pivot to any merged change.  Key word, merged.

Reply:
I moved the JWT to subsys.  This was required due to the MBEDTLS dependency.  It has to be added after /ext directory